server/src/main/java/org/opensearch/action/admin/cluster/state/term/GetTermVersionResponse.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.admin.cluster.state.term;
+
+import org.opensearch.cluster.ClusterName;
+import org.opensearch.cluster.ClusterState;
+import org.opensearch.core.action.ActionResponse;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+
+import java.io.IOException;
+
+/**
+ * Response object of cluster term
+ *
+ * @opensearch.internal
+ */
+public class GetTermVersionResponse extends ActionResponse {
+
+    private final ClusterName clusterName;
+    private final String clusterUUID;
+    private final long term;
+    private final long version;
```

Comment:
Might be good to encapsulate these properties into a `ClusterStateIdentifier` class

Reply:
Introduced a new Class `ClusterStateTermVersion` for usage here.